README.md

```diff
@@ -201,6 +201,17 @@ Translations of the guide are available in the following languages:
     some_function(rest, opts)
   end
   ```
+* <a name="parentheses-pipe-operator"></a>
+  Use parentheses for one-arity functions when using the pipe operator(`|>`).
```

Comment:
Minor point, I'd add a space between between "operator" and the opening parenthesis, here: `operator␣(`.